.github/workflows/build.yml

```diff
@@ -113,6 +157,18 @@ jobs:
           gpg --quiet --batch --yes --symmetric --cipher-algo AES256 --passphrase="$SETUP_ASSETS_PASSPHRASE" tModLoader.zip
           echo ::endgroup::
           
+          echo ::group::Giving Purpose         
+          if [ $BranchName = '1.4-stable' ] ; then
```

Comment:
Where is branch name defined?
This is probably the cause of build type bug I have

Reply:
it looks like in the action you can use `github.ref == 'refs/heads/<branchname>'` for branch checking. `BranchName` variable inside msbuild is defined in `src/WorkspaceInfo.targets` which is populated when you launch the setup tool